config/config.go

```diff
@@ -0,0 +1,84 @@
+package config
+
+import (
+	"errors"
+	"github.com/joho/godotenv"
+	"github.com/rs/zerolog/log"
+	"os"
+	"strconv"
+)
+
+type config struct {
+	sqlDBHost     string
+	sqlDBPort     int
+	sqlDBDatabase string
+	sqlDBUsername string
+	sqlDBPass     string
+}
+
+var conf config
+
+func LoadConfiguration() error {
+
+	c := &config{}
+
+	err := godotenv.Load()
+	if err != nil {
+		log.
+			Error().
+			Msg("Environment variables not found.")
+	}
+
+	errs := make([]string, 0)
+
+	c.sqlDBHost = os.Getenv("DB_HOST")
```

Comment:
this strings ("DB_HOST") should be in a conts, in case you reuse the name  and need to modify, just modify the variable.